app/api/search/search.js

```diff
@@ -852,6 +852,21 @@ const search = {
     const templates = await templatesModel.get();
     return updateMapping(templates);
   },
+
+  async countPerTemplate(language) {
+    const queryBuilder = documentQueryBuilder()
+      .language(language)
+      .includeUnpublished()
```

Comment:
While this, apparently, replicates the original functionality... if you have hundreds of published and unpublished entities, wouldn't this show the 'wrong' amount of available extra options for unlogged users?

Reply:
@RafaPolit Actually this is not the case, the permissions feature takes care of that on the documentQueryBuilder.